build.gradle.kts

```diff
@@ -0,0 +1,48 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "2.5.5"
+    id("io.spring.dependency-management") version "1.0.11.RELEASE"
+    kotlin("jvm") version "1.5.31"
+    kotlin("plugin.spring") version "1.5.31"
+    id("org.jlleitschuh.gradle.ktlint") version "10.0.0" apply true
+    id("jacoco")
+}
+
+group = "com.bookmarks"
+version = "0.0.1-SNAPSHOT"
+java.sourceCompatibility = JavaVersion.VERSION_11
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter:2.5.5")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    implementation("org.junit.jupiter:junit-jupiter:5.8.1")
+    implementation("org.jacoco:org.jacoco.core:0.8.7")
```

Comment:
Wow, are you actually somehow using `jacoco` inside the project? Because otherwise, it means you are bundling test-coverage library inside the application (and that is really bad)

Reply:
5699e74